### DISCUSSION TITLE
how do i steal disability modchart from golden apple edition (KE 1.2) source code?

### DISCUSSION THREAD

**Discussion by eRobLex** [KadeArchive/Kade-Engine#2828] [Q&A]
i really wanna know how!

**Comment by ghost** [comment]
Not sure why the hell you want this, but here is the link for the golden apple source code.
https://github.com/ItzEpicstickman/VsEvilCorruptedDaveDay3Final
Also stealing is bad, your bad, and anyone who mentions or does any type of stealing is bad.
Wait a minute...

**Reply by eRobLex** [reply]
big bruh. i already know that i just want to know how to take modchart (Im making a port jesus) also i stopped making the port because i noticed they said that no ports for psych engine or mic'd engine or other engines. so i dont need it anyway now

**Reply by ghost** [reply]
Oh.
Well, I still found out how to do it.
This is the line found in the source code:
```
playerStrums.forEach(function(spr:Strum)
{
	spr.angle += (Math.sin(elapsedtime * 2.5) + 1) * 5;
});
```
So you can just add this in the mod chart.
```
function onUpdate(elapsed)
	for i=0,7 do
		local receptor = _G['receptor_'..i] -- Just to make it feel easier for me lol.
		receptor.angle = receptor.angle + (Math.sin(elapsedtime * 2.5) + 1) * 5;
	end
end
```
And no I'm not dumb I do know you said you don't need it anymore. And besides, the creators never said whether the port had to be public or not. Just make a video or something.
_lol_
